PsychSourceGL/Cohorts/FTGLTextRenderer/OGLFT.cpp

```diff
@@ -3423,6 +3474,7 @@ QString Face::format_number ( const QString& format, double number ) { return( Q
 		    -rotation_offset_y_, 0.f );
     }
 
+    glBindTexture( GL_TEXTURE_2D, texture_info.texture_name_ );
```

Comment:
Why move this down from its position above?  Could/Should stay where it was.

Reply:
Will test with it back in place, probably a left-over from many things i tried